### DISCUSSION TITLE
zigbee-herdsman-converters version

### DISCUSSION THREAD

**Discussion by erdosip** [Koenkk/zigbee2mqtt#14039] [General]
Hello,

I would like to ask about the versioning of zigbee-herdsman-converters.
I have two working zigbee2mqtt instance, which is installed from the git repository, and sometimes I run the update.sh to get the newest stuff.

While doing my business, I just realized, that the device capabilities has to came from zigbee-herdsman-converters.
AFAIK the package.json force this  to the specific version:   "zigbee-herdsman-converters": "14.0.610"

In the converter repository, there are newer tags (14.0.625 is the newest) 
Is there any reason, why zigbee2mqtt shipped with .610? Is there any way to upgrade mine locally (without screwing up the update process)

Thanks!

Regards, 
 Peter

**Comment by Koenkk** [comment]
The release version will always use a fixed version (to make sure nothing breaks). Dev version always uses the latest zigbee-herdsman-conveters.  (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)
